src/containers/pages/MDAPoint/ClientUpload/tests/index.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { cleanup } from '@testing-library/react';
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import { Router } from 'react-router';
+import ClientUpload from '..';
+const history = createBrowserHistory();
+
+describe('components/ClientUpload', () => {
```

Comment:
Can we make this test better?  You just have one snapshot test here.  You aren't testing how the component works.

Reply:
https://github.com/onaio/reveal-frontend/issues/889